indico/modules/events/papers/forms.py

```diff
@@ -145,3 +145,12 @@ def __init__(self, *args, **kwargs):
         if template is None:
             inject_validators(self, 'template', [DataRequired()])
         super(PaperTemplateForm, self).__init__(*args, **kwargs)
+
+
+class DeadlineForm(IndicoForm):
+    deadline = IndicoDateTimeField(_("Deadline"), [Optional()], default_time=time(23, 59))
+    enforce = BooleanField(_("Enforce deadline"), default=True)
```

Comment:
switch widget please, and the `default=True` shouldn't be needed since you always provide the current settings when rendering the form